apps/framework-docs/src/pages/quick-reference.mdx

```diff
@@ -0,0 +1,341 @@
+import { Tabs } from "nextra/components";
+
+# Quick Reference
+
+This is a quick guide on using MooseJS with all the basic code in one place. For more details you can visit individual pages on each section.
+
+---
+
+<Tabs items={["Data Models", "Flows", "Aggregations", "Consumption APIs"]}>
+
+    <Tabs.Tab>
+        ## Creating a Model
+
+        To create a data model you need to open the `models.ts` file in your project's `datamodels` directory
+        and add a new Data Model definition.
+
+        ```ts filename="datamodels/models.ts" copy
+        import { Key } from "@514labs/moose-lib";
+
+        export interface UserActivity {
+        eventId: Key<string>;
+        timestamp: string;
+        userId: string;
+        activity: string;
+        }
+        ```
+
+        ## Supported Types
+
+        Here is the status of MooseJS support for various data types in Data Models:
+        | Clickhouse | Typescript | Moose |
+        |:---------- |:---------- |:------- |
+        | String | String | ✅ |
+        | Boolean | Boolean | ✅ |
+        | Int64 | Number | ✅ |
```

Comment:
I think it's misleading to say we support other number types as we map all ts number types to Float64

Reply:
I have just added a disclaimer to the quick reference page and the data models setup page in the latest PR.